Modules/posixmodule.c

```diff
@@ -5535,15 +5535,16 @@ os__path_lexists_impl(PyObject *module, path_t *path)
 /*[clinic input]
 os._path_isdir -> bool
 
-    s as path: path_t(allow_fd=True, suppress_value_error=True)
+    s as path: path_t(allow_fd=True, suppress_value_error=True),
```

Comment:
I suggest to change also the parameter name.
```suggestion
    path: path_t(allow_fd=True, suppress_value_error=True),
```